### PR TITLE
Feature/run without digitizer

### PR DESCRIPTION
**Pull request by MCFlowMace**

### CODE REVIEW

File: Source/Core/LMCEggWriter.cc [hunk 1]
```diff
@@ -106,34 +106,45 @@ namespace locust
...
-            t_signed_vals = a_digitizer->GetADCValuesSigned();
+            t_data_format = a_digitizer->GetADCValuesSigned();
             t_bit_depth = a_digitizer->DigitizerParams().bit_depth;
             t_bits_right_aligned = a_digitizer->DigitizerParams().bits_right_aligned;
-        }
+            t_v_range = a_digitizer->DigitizerParams().v_range;
+            t_v_offset = a_digitizer->DigitizerParams().v_offset;
+            t_dac_gain = a_digitizer->DigitizerParams().dac_gain;
+        } else {
+			LDEBUG( lmclog, "Running without digitizer, data saved in float format");
```
Comment (pslocum) [inline Source/Core/LMCEggWriter.cc:127]: Consider upgrading from LDEBUG to LWARN.  Probably not very important.

**Review by pslocum — COMMENTED** [review]
These changes look straightforward and produce normal-looking output for default configs.  What I can't find is how to actually test the new feature.  Is there a parameter that I need to type?

**Comment by MCFlowMace** [comment]
> These changes look straightforward and produce normal-looking output for default configs. What I can't find is how to actually test the new feature. Is there a parameter that I need to type?

With the changes you can just omit `digitizer` from the `generators` list. No new keyword was added.

**Comment by MCFlowMace** [comment]
> These changes look straightforward and produce normal-looking output for default configs. What I can't find is how to actually test the new feature. Is there a parameter that I need to type?

With the changes you can just omit `digitizer` from the `generators` list. No new keyword was added.

**Comment by pslocum** [comment]
This seems to run smoothly and I think it looks ready to be merged, unless we should wait for additional changes?

**Comment by MCFlowMace** [comment]
I do not have any additional changes planned at the moment. We can go ahead from my side.

**Pull request merged**